web/crux/src/app/agent/agent.service.ts

```diff
@@ -447,7 +451,7 @@ export default class AgentService {
     })
   }
 
-  private async updateDeploymentStatuses(nodeId: string, deploymentId: string, status: DeploymentStatus) {
+  private async updateDeploymentStatuses(nodeId: string, deploymentInstance: Deployment, status: DeploymentStatus) {
```

Comment:
why we need the whole instance?

Reply:
To have a reference to the `mergedConfigs` which is inside the deployment.